capz/run-capz-e2e.sh

```diff
@@ -503,7 +508,7 @@ log() {
 
 # all test regions must support AvailabilityZones
 get_random_region() {
-    local REGIONS=("australiaeast" "canadacentral" "eastus" "eastus2" "northeurope" "uksouth" "westus2" "northcentralus")
+    local REGIONS=("australiaeast" "canadacentral" "francecentral" "germanywestcentral" "northeurope" "switzerlandnorth" "uksouth")
```

Comment:
I hit some inconsistencies with image replication across the different regions, and locked to `northcentralus` till I'd had a chance to review it.

Reply:
CAPZ maintains a list here: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/71a3924a3150e38aa1dee93bc0bb7dec57d75943/hack/util.sh#L65